rhv_checks.py

```diff
@@ -338,6 +340,54 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the host's Hosted Engine Status."""
+    okay, critical, unknown, all_items = [], [], [], []
+    command = "hosted-engine --vm-status --json"
+    hosts = system.api.system_service().hosts_service().list()
+    output = ssh_command(hosts[0].name, password, command)
+    # Output is in the format of List of JSON removing newline.
+    if len(output) != 0:
+        result = output[0].replace("\\n", "").replace("\n", "")
+        host_data_dict = json.loads(result)
+        # Iterating value from Dict --> Dict --> Value.
+        for host_index, host_detail in host_data_dict.items():
+            if host_index == 'global_maintenance':
+                continue
+            try:
+                health = host_detail['engine-status']['health']
```

Comment:
@john-dupuy @sbulage One of the other things I wanted to check in this is that the host is able to run the hosted-engine.  I don't see any of that here. Please review my comments https://github.com/kedark3/check-rhv/pull/5#discussion_r289097552
Did you guys had any discussion around that to be excluded for any reason?

Reply:
Nope, I must have missed that comment, @sbulage do you want that to be excluded?

Reply:
Added `for-loop` there which iterate through all hosts, and gives unique `result` which then updated in `output`.